te/RQ_CONNECTOR.py

```diff
@@ -158,7 +158,7 @@ def start_stat_collector(self):
         stat_collector_config = {
             "te_ip" : self.ctrl_ip,
             "my_ip" : self.ip,
-            "self.stat_collect_interval" : int(self.stat_collect_interval),
```

Comment:
Typo in setting up stat_collector config. This resulted in stats getting collected only every 15 seconds (default) and not as per configuration.